kaisen/kaisen/src/raliat/p.py

```diff
@@ -0,0 +1,5 @@
+def subtract(a, b):
+    return a - b
+def add(a, b):
+    return a + b - subtract(a, b)
+print(subtract(121, 74))
```

Comment:
this is great work!

Reply:
Thank you !!!

Reply:
it should be `print(add(121, 74))` not `print(subtract(121, 74))`